### PR TITLE
Docs (GraphQL Subscriptions): Rewording for style, plus gzip compression support

### PR DESCRIPTION
**Pull request by aaroncarey**
Per DOC-189

### CODE REVIEW

File: content/graphql/subscriptions/index.md [hunk 1]
```diff
@@ -108,23 +108,27 @@ const wsLink = new WebSocketLink({
     connectionParams: {  "Authorization": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2OTAxMjg2MjIsImh0dHBzOi8vZGdyYXBoLmlvIjp7IlJPTEUiOiJVU0VSIiwiVVNFUiI6IkFsaWNlIn0sImlzcyI6InRlc3QifQ.6AODlumsk9kbnwZHwy08l40PeqEmBHqK4E_ozNjQpuI", },});
 ```
 
-### Working
+{{% notice "note" %}}
```
Comment (aaroncarey) [inline content/graphql/subscriptions/index.md:111]: This note is per DOC-189

File: content/graphql/subscriptions/index.md [hunk 2]
```diff
@@ -108,23 +108,27 @@ const wsLink = new WebSocketLink({
...
+Starting in release v21.03, Dgraph supports gzip compression for subscriptions.
+Dgraph uses gzip compression if the GraphQL client's `Sec-Websocket-Extensions`
```
Comment (minhaj-shakeel) [inline content/graphql/subscriptions/index.md:113]: replace gzip with `permessage-deflate` compression.

File: content/graphql/subscriptions/index.md [hunk 3]
```diff
@@ -108,23 +108,27 @@ const wsLink = new WebSocketLink({
     connectionParams: {  "Authorization": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2OTAxMjg2MjIsImh0dHBzOi8vZGdyYXBoLmlvIjp7IlJPTEUiOiJVU0VSIiwiVVNFUiI6IkFsaWNlIn0sImlzcyI6InRlc3QifQ.6AODlumsk9kbnwZHwy08l40PeqEmBHqK4E_ozNjQpuI", },});
 ```
 
-### Working
+{{% notice "note" %}}
+Starting in release v21.03, Dgraph supports gzip compression for subscriptions.
```
Comment (minhaj-shakeel) [inline content/graphql/subscriptions/index.md:112]: Dgraph supports compression for .....

**Review by minhaj-shakeel — CHANGES_REQUESTED** [review]

File: content/graphql/subscriptions/index.md (hunk 2 above)
Comment (aaroncarey) [reply content/graphql/subscriptions/index.md]: Done.

File: content/graphql/subscriptions/index.md (hunk 3 above)
Comment (aaroncarey) [reply content/graphql/subscriptions/index.md]: Done.

**Comment by aaroncarey** [comment]
Feedback incorporated, merging

**Pull request merged**